collectd.env

```diff
@@ -0,0 +1,23 @@
+# if monitoring only one MySQL server
+MYSQL_SERVER=localhost
```

Comment:
Can we rename it to MYSQL_NAME or MYSQL_ALIAS or ...? MYSQL_SERVER may be confusing and doesn't tell what it is.